src/relative-time-element.ts

```diff
@@ -382,6 +383,14 @@ export class RelativeTimeElement extends HTMLElement implements Intl.DateTimeFor
     this.setAttribute('format-style', value)
   }
 
+  get noTitle(): boolean {
+    return this.hasAttribute('no-title')
+  }
+
+  set noTitle(value: boolean | undefined) {
+    this.toggleAttribute('no-title')
```

Comment:
Consider providing a [`force` parameter](https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute#force):

```suggestion
    this.toggleAttribute('no-title', value)
```

Otherwise, we’ll get surprising behavior like:
1. Initial state: `<relative-time no-title="true" />`.
2. Then, run `relativeTimeElement.noTitle = true`
3. New state: `<relative-time />`.

Reply:
Ah, another thing I didn't know existed! Thank you!